lib/route.js

```diff
@@ -955,6 +955,23 @@ function route(harvester, name, model, schema, routeOptions) {
     }
 
     this.appendLinks = appendLinks;
+
+    function disallow(methods) {
+        _.forEach(methods, function (methodName) {
+            var method = _this[methodName]();
+            method.options.notAllowed = true;
+            method.register();
```

Comment:
Not sure if disallow should automatically call register... I think the intention was that register would still need to be called